isis/src/base/apps/stats/stats.xml

```diff
@@ -11,6 +11,46 @@
       processed. The <i>stats</i> program generates statistics that will help you find new insights
       in your data to make more accurate predictions and achieve better outcomes.
     </p>
+    <h5>Histogram Computation</h5>
+    <p>
+      Stats uses a histogram to compute the mode and median <def link="DigitalNumber">DN</def> values for a given cube.
+      The minimum, maximum, and number of bins depends on the pixel type of the cube. Here are the minimum, maximum,
+      and number of bins associated with supported pixel types:
+      <table border="1">
+        <tr border="1">
+          <th>Pixel Type</th>
+          <th>Minimum</th>
+          <th>Maximum</th>
+          <th>Number of Bins</th>
+        </tr>
+        <tr border="1">
+          <td>Unsigned Byte</td>
+          <td>0 * cube multiplier + base</td>
+          <td>255 * cube multiplier + base</td>
+          <td>256</td>
+        </tr>
+        <tr border="1">
+          <td>Unsigned Word</td>
+          <td>0 * cube multiplier + base</td>
+          <td>65535 * cube multiplier + base</td>
+          <td>65536</td>
+        </tr>
+        <tr border="1">
+          <td>Signed Word</td>
+          <td>-32768.0 * cube multiplier + base</td>
+          <td>32767.0 * cube multiplier + base</td>
+          <td>65536</td>
+        </tr>
```

Comment:
Even though they are the same, I suggest putting each on its own line. It's just easier to find

Reply:
@scsides Each what exactly? This is how it looks on the docs page, I'm just not sure what you are referring to.
<img width="1100" alt="Screen Shot 2020-09-01 at 11 46 33 AM" src="https://user-images.githubusercontent.com/17728237/91893456-fd518500-ec48-11ea-9cd7-b58159fc008a.png">

Reply:
Test

Reply:
The bottom row of the table combines "Unsigned Integer/Signed Integer/Real". I think it crowds the table and makes finding Real or Signed Integer difficult. My suggestion is to put each of these in their own row.